.template-helpers/variant-windowsservercore.md

```diff
@@ -0,0 +1,8 @@
+## `%%REPO%%:windowsservercore`
+
+This image is based on [Windows Server Core (`microsoft/windowsservercore`)](https://hub.docker.com/r/microsoft/windowsservercore/). As such, it only works in places which that image does, such as Windows 10 (with insider previews) or Windows Server 2016.
+
+For information about how to get "Docker on Windows" installed and configured, please see the relevant "Quick Start" guide provided by Microsoft:
```

Comment:
Nit but might be a tiny bit clearer (as I always end up reading it as `Docker for Windows` which is obviously something quite different) if you put ...'about how to get Docker running on Windows, please see the relevant'...

@neilpeterson is our docs guy, but I think those links are OK :innocent: